### DISCUSSION TITLE
403 Forbidden Errors Due To Connected App Scopes or Limitations?

### DISCUSSION THREAD

**Discussion by GoBenGoh** [mulesoft-catalyst/metrics-toolkit#177] [Q&A]
I'm getting 403 forbidden errors for HTTP GET resource on these links:

- https://anypoint.mulesoft.com/cloudhub/api/v2/applications
- https://anypoint.mulesoft.com/apimanager/api/v1/organizations/.../environments/.../apis
- https://anypoint.mulesoft.com/apimanager/api/v1/organizations/.../automated-policies
- https://anypoint.mulesoft.com/hybrid/api/v2/organizations/.../environments/.../deployments
- https://anypoint.mulesoft.com/hybrid/api/v1/clusters
- https://anypoint.mulesoft.com/hybrid/api/v1/serverGroups

Would this be due to a lack of scopes for my Connected App or are they part of the limitations of authenticating with a CA?

**Comment by richardmckinley** [comment]
Hi @GoBenGoh 

The limitations of using a connected app are documented in the [Limitations](https://github.com/mulesoft-catalyst/metrics-toolkit#limitations) section. 
It however looks more like a lack of scopes associated with your CA. Please ensure the app has all required scopes defined in the [Requirements](https://github.com/mulesoft-catalyst/metrics-toolkit#requirements) section, and in all required business groups and environments. 

Thanks, 
Richard
